db/watchdog.c

```diff
@@ -435,7 +435,7 @@ void watchdog_enable(void)
 
 void lock_info_lockers(FILE *out, bdb_state_type *bdb_state);
 
-void comdb2_die(int aborat)
+void comdb2_die(int doabort, const char *str)
```

Comment:
Since you're not using `str` here at all, maybe void it - some compilers do warn of it.